src/features/Classes/ClassesTable/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { IntlProvider } from 'react-intl';
+import { Row, Col } from '@edx/paragon';
+import DataTable from '@edx/paragon/dist/DataTable';
```

Comment:
```suggestion
import { Row, Col, Datatable } from '@edx/paragon';
```
Move Datable import with others

Reply:
@AuraAlba I got and error if I try to do this.